libmscore/scale.h

```diff
@@ -0,0 +1,107 @@
+//=============================================================================
+//  MuseScore
+//  Music Composition & Notation
+//
+//  Copyright (C) 2002-2012 Werner Schweer
+//
+//  This program is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License version 2
+//  as published by the Free Software Foundation and appearing in
+//  the file LICENCE.GPL
+//=============================================================================
+
+#ifndef __SCALE_H__
+#define __SCALE_H__
+
+/**
+ \file
+ Definition of Scale class.
+ */
+
+#include "note.h"
+
+namespace Ms {
+
+struct ScaleParams {
+      QString notes[TPC_NUM_OF];
+      int     nbNotes;
+      int     storingMode;
+      int     reference;
+      bool    storeFifths;
+};
+
```

Comment:
As I need to convert between different storing modes (for the combos in the editors), I didn't want to have functions with a lot of parameters so I put all of them in one struct.